src/ast.ls

```diff
@@ -1641,14 +1641,17 @@ class exports.Assign extends Node
         else if (ret or len > 1) and (not ID.test rite.to-string! or left.assigns rite.to-string!)
             cache = sn(this, (rref = o.scope.temporary!), " = ", rite)
             rite  = rref
-        list = @"rend#{ left.constructor.display-name }" o, items, rite
+        if rite.to-string! is \arguments and not ret
+            arg = true
```

Comment:
better (refactorable) names on the way; I would rather not push before I fix the named nested splat though.